src/EnrollChoicesController.js

```diff
@@ -10,18 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint complexity: [2, 13] */
+/* eslint complexity: [2, 14] */
 define([
   'okta',
   './util/FormController',
   './util/Enums',
   './util/RouterUtil',
   'views/enroll-choices/FactorList',
   'views/enroll-choices/RequiredFactorList',
-  'views/enroll-choices/Footer'
+  'views/enroll-choices/Footer',
+  'views/shared/FooterWithBackLink'
 ],
 function (Okta, FormController, Enums, RouterUtil, FactorList,
-  RequiredFactorList, Footer) {
+  RequiredFactorList, Footer, FooterWithBackLink) {
```

Comment:
If Footer also implements customBackLinkInMFA, why do we need FooterWithBackLink?

Reply:
`enroll-factors/Footer` supports custom back link, but in here we use `enroll-choices/Footer, which only supports a skip link.